core/src/main/resources/db/changelog/changesets/2024_03_29_portal_scoped_stable_ids.yaml

```
@@ -0,0 +1,76 @@
+databaseChangeLog:
+  - changeSet:
+      id: "portal_scoped_stable_ids"
+      author: connorlbark
+      changes:
+        - dropForeignKeyConstraint:
```

I suspect they're funcitonally identical, but for hygiene
```suggestion
        - dropUniqueConstraint:
```